coordinator/src/model/fhir-resources.ts

```diff
@@ -260,3 +260,16 @@ export interface ContentReference {
   reference: string
   display: string
 }
+
+interface BackboneElement {
```

Comment:
BackboneElement is a generic term used in the FHIR spec for something that doesn't have a proper name. I'd call this class MessageHeaderSource instead